Formula/pgloader.rb

```diff
@@ -3,8 +3,8 @@ class Pgloader < Formula
   homepage "https://github.com/dimitri/pgloader"
   url "https://github.com/dimitri/pgloader/archive/v3.2.2.tar.gz"
   sha256 "5fe5c115e277a9dd616b1077f89bffdf978bc6983ce62d99af9a218142c39e40"
-  head "https://github.com/dimitri/pgloader.git"
   revision 1
```

Comment:
insert a blank line here because we have a revision. see comment on libhttpserver